lib/v2/hit/hitgs.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const host = 'http://hitgs.hit.edu.cn';
+
+    const response = await got({
+        method: 'get',
+        url: host + '/tzgg/list.htm',
+        headers: {
+            Referer: host,
+        },
+    });
+
+    const $ = cheerio.load(response.data);
+    const list = $('.news_list li')
+        .map((i, e) => ({
+            path: $('span a', e).attr('href'),
+            pubDate: new Date($('span:nth-child(4)', e).text()).toUTCString(),
```

Comment:
Please use `parseDate`
docs: https://docs.rsshub.app/joinus/pub-date.html
```suggestion
            pubDate: parseDate($('span:nth-child(4)', e).text()),
```